packages/studio/src/components/PageSettingsButton/EntityPageModal.tsx

```diff
@@ -9,13 +9,32 @@ import StreamScopeParser, {
 } from "../../utils/StreamScopeParser";
 import { PageSettingsModalProps } from "./PageSettingsButton";
 import { StaticPageSettings } from "./StaticPageModal";
-import { streamScopeFormData } from "../AddPageButton/StreamScopeCollector";
 import PageDataValidator from "../../utils/PageDataValidator";
 import { toast } from "react-toastify";
 import { isEqual } from "lodash";
 
 type EntityPageSettings = StaticPageSettings & StreamScopeForm;
 
+export const streamScopeFormData: FormData<StreamScopeForm> = {
```

Comment:
Made an item for the Edit Flow piece: https://yexttest.atlassian.net/browse/SLAP-2918. Could we add a TODO in this file?